kafka-streams/src/main/java/io/confluent/developer/aggregate/StreamsAggregate.java

```diff
@@ -39,7 +39,12 @@ public static void main(String[] args) throws IOException {
 
         // Now take the electronicStream object, group by key and perform an aggregation
         // Don't forget to convert the KTable returned by the aggregate call back to a KStream using the toStream method
-        electronicStream.groupByKey().aggregate(null, null);
```

Comment:
These tutorials were intentionally left incomplete, so users can finish the code on their own.  Full solutions are found in the `solution` package for each tutorial.

Reply:
Yep sorry, I pushed to my main branch after completing the course and the PR auto-updated to the latest commit.